src/xcode/ENA/ENA/Source/Scenes/TestResultAvailable/TestResultAvailableViewController.swift

```diff
@@ -58,7 +61,17 @@ final class TestResultAvailableViewController: DynamicTableViewController, ENANa
 	private func setupTableView() {
 		view.backgroundColor = .enaColor(for: .background)
 		tableView.separatorStyle = .none
-		dynamicTableViewModel = viewModel.dynamicTableViewModel
+	}
+
+	private func setupViewModel() {
+		viewModel.$dynamicTableViewModel.sink { [weak self] dynamicTableViewModel in
+			guard let self = self else {
+				Log.debug("Faile to get dynamic tableview model - return early")
```

Comment:
You could remove this guard and just use `self?.`, only if you want to 😄 But the log is outdated and should be removed 😁